### DISCUSSION TITLE
Env management

### DISCUSSION THREAD

**Discussion by samchouse** [prisma/prisma#6278] [Q&A] [answered]
Any alternatives to #2310?

**Comment by leo-petrucci** [comment, marked as answer]
The only alternative I can think of would be setting variables directly from the command:
```
MY_ENV_VAR=something next build
```

**Reply by ryands17** [reply]
Why not use `dotenv-cli` as mentioned in the thread? That would make handling envs much easier.

**Reply by samchouse** [reply]
I just find it inefficient.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
